src/features/auth/authService.test.js

```diff
@@ -227,28 +224,30 @@ describe('authService', () => {
     };
 
     beforeEach(() => {
-      axios.get.mockReset();
+      axios.mockReset();
 
-      axios.get.mockResolvedValue({
+      axios.mockResolvedValue({
         data: {
-          firstToken: '71ffd52c-5985-46d3-b445-490554f4012a',
-          secondToken: 'de7c2a83-07f2-46bf-8417-8f648adbc7be',
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          access_token: '71ffd52c-5985-46d3-b445-490554f4012a',
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          refresh_token: 'de7c2a83-07f2-46bf-8417-8f648adbc7be',
         },
       });
     });
-    it('should call axios.get with the right arguments', async () => {
+    it('should call axios.post with the right arguments', async () => {
       expect.assertions(2);
-      await authService.fetchApiTokens(mockUser);
+      await authService.fetchApiToken(mockUser, 'berths-api');
 
-      expect(axios.get).toHaveBeenCalledTimes(1);
-      expect(axios.get.mock.calls[0]).toMatchSnapshot();
+      expect(axios).toHaveBeenCalledTimes(1);
+      expect(axios.mock.calls[0]).toMatchSnapshot();
     });
```

Comment:
Just curious: does `axios.mock` now mean `axios.post.mock` in a way? As you've changed the test description to `axios.post` and the code to `axios.mock` from `axios.get.mock`?

Reply:
axios have 3 different ways it can be used:
- axios.get - shortcut for get requests, with limited options available.
- axios.post - shortcut for post requests, with limited options available.
- axios - client with all the options. Then the method must be given as an option.

Reply:
There are no new snapshot values and the existing ones have not been changed, so is `expect(axios.get.mock.calls[0]).toMatchSnapshot();` → `expect(axios.mock.calls[0]).toMatchSnapshot();` correct if the snapshot content is the same and the method must be given to `axios` as an option? Or is it just that the snapshots have not been updated?